src/Badge/Model/UseCase/CreateComposerLockBadge.php

```diff
@@ -64,8 +64,8 @@ public function createComposerLockBadge(string $repository, string $format = 'sv
             'HEAD',
             $repo.'/blob/'.$package->getDefaultBranch().'/composer.lock',
             [
-                RequestOptions::TIMEOUT => 2,
-                RequestOptions::CONNECT_TIMEOUT => 1,
+                RequestOptions::TIMEOUT => 8,
```

Comment:
Could it be better to put these values into constants?
So we could also specify if they are seconds or other to improve readability, what do you think?

Reply:
make sense...
What name would you give to constants?

Reply:
For example something like 
```
TIMEOUT_SECONDS
CONNECT_TIMEOUT_SECONDS
```

make sense?

Reply:
yep!